wa/workloads/gfxbench/__init__.py

```diff
@@ -0,0 +1,61 @@
+#    Copyright 2014-2016 ARM Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import re
+
+from wa import ApkUiautoWorkload, WorkloadError
+
+
+class Gfxbench(ApkUiautoWorkload):
+
+    name = 'gfxbench'
+    package_names = ['net.kishonti.gfxbench.gl.v50000.corporate']
```

Comment:
Is this specific to the corporate (in which case, the workload should be renamed to reflect that), or will this work with the free version (in which case, its package should be added here)?